packages/admin/admin/src/dataGrid/renderCellUsingValueOptions.ts

```diff
@@ -0,0 +1,19 @@
+import { GridColDef, ValueOption } from "./GridColDef";
+
+export const renderCellUsingValueOptions: GridColDef["renderCell"] = ({ value, colDef }) => {
```

Comment:
I'm not fond of this name, but I don't have a better option. Maybe @nsams can provide one.

Reply:
- renderCellUsingStaticSelect
- renderStaticSelectCell
- renderStaticSelect
- renderCellStaticSelect

Alternative: we provide "[custom columns](https://mui.com/x/react-data-grid/custom-columns/)":
```
        {
            ...staticSelectColumn,
            field: "inStock",
            headerName: intl.formatMessage({ id: "product.inStock", defaultMessage: "In Stock" }),
            valueOptions: [...]
        }
```

(with the plan to have more such Columns, for example for date/dateTime - @Ben-Ho is working on that)

Reply:
I like `renderStaticSelectCell` 👍🏼 

Should we just use this for now? 
We can always add the custom columns later.

Reply:
The thing with naming is, that our package is named `@comet/admin`, not `@comet/admin-datagrid` or so. This means we always have to include the DataGrid association somehow in the names. Question is, if `....Cell` is enough. Maybe yes.

> We can always add the custom columns later.

:+1: